aws/resource_aws_acm_certificate.go

```diff
@@ -137,19 +203,21 @@ func resourceAwsAcmCertificateRead(d *schema.ResourceData, meta interface{}) err
 			return resource.NonRetryableError(err)
 		}
 
-		domainValidationOptions, emailValidationOptions, err := convertValidationOptions(resp.Certificate)
+		certificateDetails, err := convertCertificateDetails(resp.Certificate)
+
+		if len(certificateDetails) < 1 {
+			return resource.NonRetryableError(fmt.Errorf("Error getting certificate details"))
+		}
 
 		if err != nil {
 			return resource.RetryableError(err)
 		}
 
-		if err := d.Set("domain_validation_options", domainValidationOptions); err != nil {
+		if err := d.Set("certificate_details", certificateDetails); err != nil {
 			return resource.NonRetryableError(err)
 		}
-		if err := d.Set("validation_emails", emailValidationOptions); err != nil {
```

Comment:
Same here. If we are going to deprecate the old attributes, they still need to be set in the state to not break backwards compatibility.